### PR TITLE
Remove Rafter from Application Connector

### PR DESCRIPTION
**Pull request by mvshao**
**Description**
The feature will be no longer used in Kyma 2.1, so we need to drop the functionality.

Changes proposed in this pull request:

- Delete all Rafter dependencies

**Related issue(s)**
#12844

### CODE REVIEW

File: components/application-registry/README.md [hunk 1]
```diff
@@ -33,9 +33,7 @@ The Application Registry has the following parameters:
 - **requestTimeout** is the time-out for requests sent through the Application Registry. It is provided in seconds. The default time-out is `1`.
 - **requestLogging** is the flag for logging incoming requests. The default value is `false`.
 - **specRequestTimeout** is the time-out for requests fetching specifications provided by the user. It is provided in seconds. The default time-out is `20`.
-- **rafterRequestTimeout** is the time-out for requests fetching specifications from Rafter. It is provided in seconds. The default time-out is `20`.
 - **detailedErrorResponse** is the flag for showing detailed internal error messages in response bodies. The default value is `false` and all internal server error messages are shortened to `Internal error`, while all other error messages are shown as usual.
-- **uploadServiceURL** is the URL of the Upload Service.
 - **insecureAssetDownload** is the flag for skipping certificate verification for asset download. The default value is `false`.
 - **insecureSpecDownload** is the flag for skipping certificate verification for API specification download. The default value is `false`.
```
Comment (akgalwas) [inline components/application-registry/README.md:38]: `insecureAssetDownload`  `insecureSpecDownload`  and It is no longer needed.

**Review by majakurcius — APPROVED** [review]

**Comment by janmedrek** [comment]
Well, that was fast, seems like all the affected files were removed...

**Pull request closed without merging**